luxonis_ml/nn_archive/config_building_blocks/base_models/head.py

```diff
@@ -117,6 +127,15 @@ def validate_label_type(
             raise ValueError("Invalid family")
         return value
 
+    @field_validator("metadata")
+    def validate_metadata(cls, value):
+        if not isinstance(value, HeadObjectDetectionMetadata):
```

Comment:
Same as above

Reply:
Thanks, removed in [1031e73](https://github.com/luxonis/luxonis-ml/pull/127/commits/1031e733652d54e98130f6baf7400c621414b5eb).